src/create_disk.sh

```diff
@@ -123,6 +123,22 @@ set prefix=($prefix)/grub2
 normal
 EOF
 
+# copy the boot binaries needed for Raspberry pi support
+if  [ -d "/usr/sihare/uboot/" ]; then
```

Comment:
Typo here; I think this gets to what I meant by using `if [ "$arch" == "aarch64" ]` as the only test - if we *always* want RPi support, then the build should fail if the file goes missing (or change names).

Reply:
I used the check for u-boot because if we support armhfp we will need to have a separate definition there. though it could be done in other ways. I have fixed the typo, I had fixed that in my test run and forgot to fix it in the checkout that I have for submitting changes